### PR TITLE
feat: add frontend option

### PR DESCRIPTION
**Pull request by denise89p**

### CODE REVIEW

File: clusterize/clusterize.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ type ClusterParams struct {
 	DataProtection    DataProtectionParams
 	InstallDpdk       bool
 	DebugOverrideCmds string
+	AddFrontend       bool
```
Comment (assafgi) [inline clusterize/clusterize.go:31]: not needed

File: clusterize/clusterize.go [hunk 2]
```diff
@@ -55,15 +56,21 @@ func (c *ClusterizeScriptGenerator) GetClusterizeScript() string {
...
-	PORTS=(14000 15000 16000)
+	CONTAINER_NAMES=(drives0 compute0)
+	PORTS=(14000 15000)
 
 	# report function definition
 	%s
 
 	# clusterize_finalization function definition
 	%s
+	if [[ $ADD_FRONTEND == true ]]; then
```
Comment (assafgi) [inline clusterize/clusterize.go:69]: should check number of frontedns

Comment (denise89p) [reply clusterize/clusterize.go:69]: i am doing convert 	
```
addFrontendNum, _ := strconv.Atoi(os.Getenv("NUM_FRONTEND_CONTAINERS"))
	addFrontend := false
	if addFrontendNum > 0 {
		addFrontend = true
	}
```

File: bash_functions/get_bash_function.go [hunk 3]
```diff
@@ -6,20 +6,48 @@ import (
 
 func GetCoreIds() string {
 	s := `
-	core_ids=$(cat /sys/devices/system/cpu/cpu*/topology/thread_siblings_list | cut -d "-" -f 1 |  cut -d "," -f 1 | sort -u | tr '\n' ' ')
-	core_ids="${core_ids[@]/0}"
-	IFS=', ' read -r -a core_ids <<< "$core_ids"
+	set -ex
+	numa_range=()
+	numa=()
+	numa_num=$(lscpu | grep "NUMA node(s):" | awk '{print $3}')
```
Comment (assafgi) [inline bash_functions/get_bash_function.go:12]: numa fix isn't part of this PR

**Pull request merged**